### ISSUE TITLE
nest.Install() call fails with file not found error

### ISSUE THREAD

**Issue by krisdamato** [nest/nest-simulator#837]
As the title says, I'm not managing to install my module which I've successfully built. Interestingly, I can install `MyModule` built in the same manner without any hiccups. 

The module is built and two `.so` files copied over to the lib path. I've added the NEST/lib/nest path to `LD_LIBRARY_PATH` too.

Any thoughts? Thanks.

**Comment by krisdamato** [comment]
Alright, I found the problem after spending the better part of a day commenting out code and rebuilding. It was a coding error, absolutely nothing to do with the build process except that the resulting library probably had some missing definitions.

For the sake of posteriority: I created a `sam_names.h` file and corresponding source file containing name definitions for my models. I was using `namespace sam` everywhere in my code, including the header file just mentioned, where I had a nested `namespace sam { namespace names {...}}`. Unfortunately, in the source file I had the namespace incorrectly specified as `namespace nest { namespace sam {...}}`. Why that would not give a compile-time/linking error but a file-not-found error in the loading module is beyond me.
